renovate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for keras image similarity training repository",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs"
+  ],
+  "schedule": [
+    "before 3am every weekday"
+  ],
+  "enabledManagers": [
+    "github-actions",
+    "pip_requirements",
```

Comment:
Looking at the logic and fact that the Dockerfile in this repo is what is setting the python package versions and not a requirements.txt

For the first pass at this config, lets pull out the Python related pieces and I'll put in another issue so we can convert that to a requirements.txt based workflow and then we can use the normal pip_requirements manager configuration once that is completed.

Reply:
Initially, I felt the same about `requirements.txt` but the jupyter notebook file is using the python libraries underneath so I thought of putting `pip_requirements` in the renovate file. 
Nevertheless, I am fine with your suggestion. Thank you for reviewing.